crt_portal/cts_forms/templates/forms/report.html

```diff
@@ -3,27 +3,61 @@
 {% block head %}
 {{ wizard.form.media }}
 {% endblock %}
-
 {% block content %}
+  <span class="usa-sr-only">
+    Current step: {% if wizard.steps.step1 == 1 %}
+      What Happened
+    {% elif wizard.steps.step1 == 2 %}
+      Where
+    {% elif wizard.steps.step1 == 3 %}
+      Who
+    {% elif wizard.steps.step1 == 4 %}
+      Details
+    {% elif wizard.steps.step1 == 5 %}
+      Contact
+    {% elif wizard.steps.step1 == 6 %}
+      Review & Submit
```

Comment:
Since we have not made the review functionality, should it just be "Submit" for now?

Reply:
Sure!

Reply:
Implemented here: c6b7d287b98add15c4183d47fa3badea72ff1d3e.